config/routes.rb

```diff
@@ -31,8 +29,14 @@
   resources :merchants, only: [:index]
 
   resources :users, only: [:create]
-  
+
+  resources :carts, only: [:create]
```

Comment:
We can hand-roll this, or consolidate the other hand-rolled routes. Works as is, but we could be more concise / DRY.